### PR TITLE
Styling for expandable section toggles

### PR DESCRIPTION
**Pull request by rolfsmeds**
Styling for the `<summary>` element (i.e. the toggle) of expandable sections:
image

### CODE REVIEW

File: dspublisher/theme/global.css [hunk 1]
```diff
@@ -292,9 +292,22 @@ html[theme~='dark'] .card.card {
...
+  cursor: pointer;
+  font-size: var(--docs-font-size-m) !important;
+}
+
+[class^='ArticleLayout'] details:not([open]) > summary.title {
+  color: var(--docs-link-color);
+}
+
+[class^='ArticleLayout'] details[open] > summary.title {
+  color: var(--color-secondary-text);
```
Comment (jouni) [inline dspublisher/theme/global.css:312]: Any reason why not use this property instead?

```suggestion
  color: var(--docs-secondary-text-color);
```

File: dspublisher/theme/global.css [hunk 2]
```diff
@@ -292,9 +292,22 @@ html[theme~='dark'] .card.card {
...
+[class^='ArticleLayout'] .guilabel {
   font-family: var(--docs-font-family-ui);
   font-size: 0.875em;
   line-height: 1.25;
   font-weight: 600;
 }
+
+[class^='ArticleLayout'] details > summary.title {
+  cursor: pointer;
+  font-size: var(--docs-font-size-m) !important;
```
Comment (jouni) [inline dspublisher/theme/global.css:304]: If we used some other class name than `title`, we could avoid this `!important`.

Personally, I might add the class on the `<details>` element instead, if we want to add a variant for it. 

That said, I would just style the raw elements, and let all of them look the same across the docs. I’m not aware of any other places where it is currently used than in [Styling components](https://vaadin.com/docs/latest/ds/customization/styling-components/#sub-components), and I think that could look the same as the new ones in the upgrade guide.

File: dspublisher/theme/global.css (hunk 1 above)
Comment (rolfsmeds) [reply dspublisher/theme/global.css:312]: Ah, didn't notice both existed.

File: dspublisher/theme/global.css (hunk 2 above)
Comment (rolfsmeds) [reply dspublisher/theme/global.css:304]: Anyhoo, removed the .title from selectors now that the prefixes make them specific enough without

Comment (jouni) [reply dspublisher/theme/global.css:304]: Oh, right, so it seems. I didn't realize there's this `[%collapsible]` macro in AsciiDoc. I used raw HTML with a passthrough block before, which doesn't add the title class name.

Comment (jouni) [reply dspublisher/theme/global.css:304]: I think we should add some nice styles for the collapsible block in DSP built-in styles.

Comment (rolfsmeds) [reply dspublisher/theme/global.css:304]: True that. Made a ticket about it https://github.com/vaadin/docs-app/issues/328

**Review by tarekoraby — APPROVED** [review]

**Pull request merged**